src/flex/FlexView.js

```diff
@@ -149,7 +150,8 @@ export default React.createClass({
   getClasses() {
     const direction = this.props.column ? 'flex-column' : 'flex-row';
     const contentAlignment = this.getContentAlignmentClasses();
-    return cx('react-flex-view', direction, contentAlignment, this.props.className);
+    const wrap = this.props.wrap && 'flex-wrap';
```

Comment:
I think this adds a class `"false"` it `this.props.wrap` is `false`/`undefined`. have you checked it?